catroid/src/main/java/org/catrobat/catroid/utils/notifications/StatusBarNotificationManager.java

```diff
@@ -119,12 +121,14 @@ public void showOrUpdateNotification(Context context, NotificationData notificat
 		int notificationID = notificationData.getNotificationID();
 		notificationData.setProgressInPercent(progressInPercent);
 		if (progressInPercent < MAX_PERCENT) {
+			ToastUtil.showSuccess(context,notificationData.getTextWorking());
 			notificationManager.notify(notificationID, notificationData.toNotification(context, CHANNEL_ID, contentIntent));
 		} else {
 			notificationData.setProgressInPercent(0);
 			notificationData.setMaxProgress(0);
 			notificationData.setAutoCancel(true);
 			notificationData.setOngoing(false);
+			ToastUtil.showSuccess(context,notificationData.getTextDone());
 			notificationManager.notify(notificationID, notificationData.toNotification(context, CHANNEL_ID, contentIntent));
 		}
 	}
```

Comment:
Please move notificationManager.notify out of the if and else because it is the same at the end of both branches: (Android Studio also recommends this change)
```suggestion
		if (progressInPercent < MAX_PERCENT) {
			ToastUtil.showSuccess(context,notificationData.getTextWorking());
		} else {
			notificationData.setProgressInPercent(0);
			notificationData.setMaxProgress(0);
			notificationData.setAutoCancel(true);
			notificationData.setOngoing(false);
			ToastUtil.showSuccess(context,notificationData.getTextDone());
		}
		notificationManager.notify(notificationID, notificationData.toNotification(context, CHANNEL_ID, contentIntent));
	}
```